---
name: 16464263_fig5
title: 'The complete mitochondrial genome of the enigmatic bigheaded turtle (Platysternon): description of unusual genomic features and the reconciliation of phylogenetic hypotheses based on mitochondrial and nuclear DNA.'
authorString: 'Parham JF, Feldman CR, Boore JL.'
pmcid: PMC1403801
pmid: '16464263'
abstract: ' BACKGROUND: The big-headed turtle (Platysternon megacephalum) from east Asia is the sole living representative of a poorly-studied turtle lineage (Platysternidae). It has no close living relatives, and its phylogenetic position within turtles is one of the outstanding controversies in turtle systematics. Platysternon was traditionally considered to be close to snapping turtles (Chelydridae) based on some studies of its morphology and mitochondrial (mt) DNA, however, other studies of morphology and nuclear (nu) DNA do not support that hypothesis. RESULTS: We sequenced the complete mt genome of Platysternon and the nearly complete mt genomes of two other relevant turtles and compared them to turtle mt genomes from the literature to form the largest molecular dataset used to date to address this issue. The resulting phylogeny robustly rejects the placement of Platysternon with Chelydridae, but instead shows that it is a member of the Testudinoidea, a diverse, nearly globally-distributed group that includes pond turtles and tortoises. We also discovered that Platysternon mtDNA has large-scale gene rearrangements and possesses two, nearly identical, control regions, features that distinguish it from all other studied turtles. CONCLUSION: Our study robustly determines the phylogenetic placement of Platysternon and provides a well-resolved outline of major turtle lineages, while demonstrating the significantly greater resolving power of comparing large amounts of mt sequence over that of short fragments. Earlier phylogenies placing Platysternon with chelydrids required a temporal gap in the fossil record that is now unnecessary. The duplicated control regions and gene rearrangements of the Platysternon mtDNA probably resulted from the duplication of part of the genome and then the subsequent loss of redundant genes. Although it is possible that having two control regions may provide some advantage, explaining why the control regions would be maintained while some of the duplicated genes were eroded, examples of this are rare. So far, duplicated control regions have been reported for mt genomes from just 12 clades of metazoans, including Platysternon.'
doi: 10.1186/1471-2148-6-11
thumb_url: 'http://europepmc.org/articles/PMC1403801/bin/1471-2148-6-11-5.gif'
figure_no: 5
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC1403801/bin/1471-2148-6-11-5.jpg' style='max-height: 300px'>
### Figure 5
<p style='font-size: 10px;'>**Mt genomic features of *Platysternon***. Typical arrangement of vertebrate mitochondrial genes including a single control region compared to that of *Platysternon*. All genes are transcribed from left to right except where indicated by arrows. The genes that are rearranged in *Platysternon *are indicated in bold while the duplicated control regions are designated by numerals (cr1, cr2). The grey boxes in the *Platysternon *genome represent non-coding regions (perhaps degraded duplicated copies of genes) that are not present in typical vertebrate mt genomes. This figure illustrates how sequences from two non-adjacent regions were inserted between *trnI *and *trnQ*.</p>