---
name: 18627633_fig2
title: 'Molecular phylogeny of diplomonads and enteromonads based on SSU rRNA, alpha-tubulin and HSP90 genes: implications for the evolutionary history of the double karyomastigont of diplomonads.'
authorString: 'Kolisko M, Cepicka I, Hampl V, Leigh J, Roger AJ, Kulda J, Simpson AG, Flegr J.'
pmcid: PMC2496913
pmid: '18627633'
abstract: " BACKGROUND: Fornicata is a relatively recently established group of protists that includes the diplokaryotic diplomonads (which have two similar nuclei per cell), and the monokaryotic enteromonads, retortamonads and Carpediemonas, with the more typical one nucleus per cell. The monophyly of the group was confirmed by molecular phylogenetic studies, but neither the internal phylogeny nor its position on the eukaryotic tree has been clearly resolved. RESULTS: Here we have introduced data for three genes (SSU rRNA, alpha-tubulin and HSP90) with a wide taxonomic sampling of Fornicata, including ten isolates of enteromonads, representing the genera Trimitus and Enteromonas, and a new undescribed enteromonad genus. The diplomonad sequences formed two main clades in individual gene and combined gene analyses, with Giardia (and Octomitus) on one side of the basal divergence and Spironucleus, Hexamita and Trepomonas on the other. Contrary to earlier evolutionary scenarios, none of the studied enteromonads appeared basal to diplokaryotic diplomonads. Instead, the enteromonad isolates were all robustly situated within the second of the two diplomonad clades. Furthermore, our analyses suggested that enteromonads do not constitute a monophyletic group, and enteromonad monophyly was statistically rejected in 'approximately unbiased' tests of the combined gene data. CONCLUSION: We suggest that all higher taxa intended to unite multiple enteromonad genera be abandoned, that Trimitus and Enteromonas be considered as part of Hexamitinae, and that the term 'enteromonads' be used in a strictly utilitarian sense. Our result suggests either that the diplokaryotic condition characteristic of diplomonads arose several times independently, or that the monokaryotic cell of enteromonads originated several times independently by secondary reduction from the diplokaryotic state. Both scenarios are evolutionarily complex. More comparative data on the similarity of the genomes of the two nuclei of diplomonads will be necessary to resolve which evolutionary scenario is more probable."
doi: 10.1186/1471-2148-8-205
thumb_url: 'http://europepmc.org/articles/PMC2496913/bin/1471-2148-8-205-2.gif'
figure_no: 2
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2496913/bin/1471-2148-8-205-2.jpg' style='max-height: 300px'>
### Figure 2
<p style='font-size: 10px;'>Maximum likelihood tree of the Hexamitinae-enteromonad clade based on SSU rRNA genes (GTR + Γ + I), rooted as per Figure 1. Statistical support – ML bootstraps/RELL bootstraps/ML distance bootstraps/Bayesian posterior probability. Isolate PYX was identical in sequence with isolate PSEUD. Isolate PYX was therefore not included in the analysis but added to the tree by hand. Bootstrap support values &lt;50% and posterior probabilities &lt;0.7 are depicted by asterisks, or not shown.</p>