---
name: 18702831_fig1
title: 'SlowFaster, a user-friendly program for slow-fast analysis and its application on phylogeny of Blastocystis.'
authorString: 'Kostka M, Uzlikova M, Cepicka I, Flegr J.'
pmcid: PMC2529323
pmid: '18702831'
abstract: ' BACKGROUND: Slow-fast analysis is a simple and effective method to reduce the influence of substitution saturation, one of the causes of phylogenetic noise and long branch attraction (LBA) artifacts. In several steps of increasing stringency, the slow-fast analysis omits the fastest substituting alignment positions from the analysed dataset and thus increases its signal/noise ratio. RESULTS: Our program SlowFaster automates the process of assessing the substitution rate of the alignment positions and the process of producing new alignments by deleting the saturated positions. Its use is very simple. It goes through the whole process in several steps: data input - necessary choices - production of new alignments. CONCLUSION: SlowFaster is a user-friendly tool providing new alignments prepared with slow-fast analysis. These data can be used for further phylogenetic analyses with lower risk of long branch attraction artifacts.'
doi: 10.1186/1471-2105-9-341
thumb_url: 'http://europepmc.org/articles/PMC2529323/bin/1471-2105-9-341-1.gif'
figure_no: 1
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2529323/bin/1471-2105-9-341-1.jpg' style='max-height: 300px'>
### Figure 1
<p style='font-size: 10px;'>**MP tree of 31 *Blastocystis *isolates and 3 outgroups based on SSU rDNA sequences**. MP tree of 31 *Blastocystis *isolates (host in brackets) and 3 outgroups, based on SSU rDNA sequences. Bootstrap support values for four tree-reconstructing methods – ML, MP, LD and MD, respectively – are shown at the nodes. The symbol "+" is used for bootstrap support 99 and higher (in case only one "+" symbol is present, all methods scored such a high support). The effect of slow-fast analysis on nodes is represented by arrow symbols in the figure. Increase of an average bootstrap support by more than 10% of one and more than one tree-reconstructing method in two datasets (BlastS3 and S2) is marked with "↑" and "↑↑", respectively. Similarly, the decrease of bootstrap support is marked with "↓" and "↓↓" at the particular nodes. Bootstraps of other nodes did not change dramatically. Except for our five new isolates (GERA3A, GERA3B, GEPA2, GECA2, KINIX2), *Blastocystis *isolates are labeled with accession numbers of their SSU rRNA gene sequences.</p>