---
name: 22151585_fig6
title: Chasing the hare - evaluating the phylogenetic utility of a nuclear single copy gene region at and below species level within the species rich group Peperomia (Piperaceae).
authorString: 'Naumann J, Symmank L, Samain MS, MÃ¼ller KF, Neinhuis C, dePamphilis CW, Wanke S.'
pmcid: PMC3252395
pmid: '22151585'
abstract: ' BACKGROUND: The rapidly increasing number of available plant genomes opens up almost unlimited prospects for biology in general and molecular phylogenetics in particular. A recent study took advantage of this data and identified a set of nuclear genes that occur in single copy in multiple sequenced angiosperms. The present study is the first to apply genomic sequence of one of these low copy genes, agt1, as a phylogenetic marker for species-level phylogenetics. Its utility is compared to the performance of several coding and non-coding chloroplast loci that have been suggested as most applicable for this taxonomic level. As a model group, we chose Tildenia, a subgenus of Peperomia (Piperaceae), one of the largest plant genera. Relationships are particularly difficult to resolve within these species rich groups due to low levels of polymorphisms and fast or recent radiation. Therefore, Tildenia is a perfect test case for applying new phylogenetic tools. RESULTS: We show that the nuclear marker agt1, and in particular the agt1 introns, provide a significantly increased phylogenetic signal compared to chloroplast markers commonly used for low level phylogenetics. 25% of aligned characters from agt1 intron sequence are parsimony informative. In comparison, the introns and spacer of several common chloroplast markers (trnK intron, trnK-psbA spacer, ndhF-rpl32 spacer, rpl32-trnL spacer, psbA-trnH spacer) provide less than 10% parsimony informative characters. The agt1 dataset provides a deeper resolution than the chloroplast markers in Tildenia. CONCLUSIONS: Single (or very low) copy nuclear genes are of immense value in plant phylogenetics. Compared to other nuclear genes that are members of gene families of all sizes, lab effort, such as cloning, can be kept to a minimum. They also provide regions with different phylogenetic content deriving from coding and non-coding parts of different length. Thus, they can be applied to a wide range of taxonomic levels from family down to population level. As more plant genomes are sequenced, we will obtain increasingly precise information about which genes return to single copy most rapidly following gene duplication and may be most useful across a wide range of plant groups.'
doi: 10.1186/1471-2148-11-357
thumb_url: 'http://europepmc.org/articles/PMC3252395/bin/1471-2148-11-357-6.gif'
figure_no: 6
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC3252395/bin/1471-2148-11-357-6.jpg' style='max-height: 300px'>
### Figure 6
<p style='font-size: 10px;'>***Phylogram of *agt1 *based on Bayesian inference***. Phylogram obtained from the Bayesian inference of the *agt1 *data set with relative substitution rates using the GTR+G+I model and posterior probabilities plotted above the branches. This phylogenetic tree is summarized in Figure 3. Clades are colored according to Figure 3. In accessions that are represented in the tree by different 'types', multiple copies were detected that are not monophyletic. A closer look to the resolution within the clades shows highly supported relationships down to the population level.</p>