---
name: 21427087_fig5
title: A structural determinant required for RNA editing.
authorString: 'Tian N, Yang Y, Sachsenmaier N, Muggenhumer D, Bi J, Waldsich C, Jantsch MF, Jin Y.'
pmcid: PMC3141254
pmid: '21427087'
abstract: 'RNA editing by adenosine deaminases acting on RNAs (ADARs) can be both specific and non-specific, depending on the substrate. Specific editing of particular adenosines may depend on the overall sequence and structural context. However, the detailed mechanisms underlying these preferences are not fully understood. Here, we show that duplex structures mimicking an editing site in the Gabra3 pre-mRNA unexpectedly fail to support RNA editing at the Gabra3 I/M site, although phylogenetic analysis suggest an evolutionarily conserved duplex structure essential for efficient RNA editing. These unusual results led us to revisit the structural requirement for this editing by mutagenesis analysis. In vivo nuclear injection experiments of mutated editing substrates demonstrate that a non-conserved structure is a determinant for editing. This structure contains bulges either on the same or the strand opposing the edited adenosine. The position of these bulges and the distance to the edited base regulate editing. Moreover, elevated folding temperature can lead to a switch in RNA editing suggesting an RNA structural change. Our results indicate the importance of RNA tertiary structure in determining RNA editing.'
doi: 10.1093/nar/gkr144
thumb_url: 'http://europepmc.org/articles/PMC3141254/bin/gkr144f5.gif'
figure_no: 5
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC3141254/bin/gkr144f5.jpg' style='max-height: 300px'>
### Figure 5
<p style='font-size: 10px;'>Paired stem regions are not sufficient for editing. (**A**) Schematic diagrams of WT and mutant (M) minigene constructs tested in oocyte editing assays. A single synonymous change was introduced into chicken Gabra1, *D. rerio* Gabra5 and *F. rubripes* Gabra1, converting a mismatch to a paired state. Thus, the mutated stemâ€“loop structures resemble the editing structural rearrangements seen in the Gabra3. The resulting minigenes were evaluated for editing efficiency using *Xenopus* nuclear injection. (**B**) Radioactively labeled RNA was injected into *X. laevis* oocyte nuclei. Injected RNAs were isolated and submitted to P1 digestion and nucleotides were separated on TLC plates. No detectable of editing activity was observed in both *Xenopus* nuclear (N) and cytoplasm (C), while very strong editing signal was observed in both control constructs (mouse Gabra3 and FLNA). (**C**) Sequencing further confirmed efficient specific editing at the I/M site in mGabra3 minigene but not in other mutants.</p>