---
name: 21427087_fig2
title: A structural determinant required for RNA editing.
authorString: 'Tian N, Yang Y, Sachsenmaier N, Muggenhumer D, Bi J, Waldsich C, Jantsch MF, Jin Y.'
pmcid: PMC3141254
pmid: '21427087'
abstract: 'RNA editing by adenosine deaminases acting on RNAs (ADARs) can be both specific and non-specific, depending on the substrate. Specific editing of particular adenosines may depend on the overall sequence and structural context. However, the detailed mechanisms underlying these preferences are not fully understood. Here, we show that duplex structures mimicking an editing site in the Gabra3 pre-mRNA unexpectedly fail to support RNA editing at the Gabra3 I/M site, although phylogenetic analysis suggest an evolutionarily conserved duplex structure essential for efficient RNA editing. These unusual results led us to revisit the structural requirement for this editing by mutagenesis analysis. In vivo nuclear injection experiments of mutated editing substrates demonstrate that a non-conserved structure is a determinant for editing. This structure contains bulges either on the same or the strand opposing the edited adenosine. The position of these bulges and the distance to the edited base regulate editing. Moreover, elevated folding temperature can lead to a switch in RNA editing suggesting an RNA structural change. Our results indicate the importance of RNA tertiary structure in determining RNA editing.'
doi: 10.1093/nar/gkr144
thumb_url: 'http://europepmc.org/articles/PMC3141254/bin/gkr144f2.gif'
figure_no: 2
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC3141254/bin/gkr144f2.jpg' style='max-height: 300px'>
### Figure 2
<p style='font-size: 10px;'>Evolutionary and structural analysis of the Gabra subunits of vertebrates. (**A**) Multiple alignments of nucleotide sequences corresponding to the edited site in Gabra3 from different species and different Gabra isoforms. Notably, the nucleotide sequences encode identical amino acid sequences. Nucleotides showing complete conservation are shaded in red, and synonymous nucleotides showing complete conservation within edited Gabra3 subunits are shaded in green. Residues altered by RNA A-to-I editing are shaded in blue, and genomic substitutions with edited residues at these positions are shaded in yellow. For abbreviations see <ext-link ext-link-type="uri" xlink:href="http://nar.oxfordjournals.org/cgi/content/full/gkr144/DC1">Supplementary Materials</ext-link>. (**B**) The degenerate nucleotide sequences encode identical amino acid sequence, where non-synonymous nucleotides (shaded in red) are largely complementary. The edited A is shaded in blue, and synonymous nucleotides are shaded in green. (**C**) Comparative analysis of the estimated equilibrium free energies (in kcal mol<sup>−1</sup>) from orthologues of different species from fish to human. The edited genes are boxed. (**D**) All known edited Gabra3 sequences are predicted to form three types of short duplex structure, with significant difference in shape of the variable region, which was represented by the sequences from *H. sapiens* (Hsa), *M. musculus* (Mmu) and *B. taurus* (Bta), respectively. A schematic picture of the duplexes is shown. (**E**) Hypothetical architecture of an edited Gabra3 duplex. The potential edited A is squared, and degenerate nucleotides are enclosed in squares. The dotted lines indicate predicted base pairs that depend on complementarities of synonymous nucleotides (G–C, A–U, G–U).</p>