---
name: 21427087_fig7
title: A structural determinant required for RNA editing.
authorString: 'Tian N, Yang Y, Sachsenmaier N, Muggenhumer D, Bi J, Waldsich C, Jantsch MF, Jin Y.'
pmcid: PMC3141254
pmid: '21427087'
abstract: 'RNA editing by adenosine deaminases acting on RNAs (ADARs) can be both specific and non-specific, depending on the substrate. Specific editing of particular adenosines may depend on the overall sequence and structural context. However, the detailed mechanisms underlying these preferences are not fully understood. Here, we show that duplex structures mimicking an editing site in the Gabra3 pre-mRNA unexpectedly fail to support RNA editing at the Gabra3 I/M site, although phylogenetic analysis suggest an evolutionarily conserved duplex structure essential for efficient RNA editing. These unusual results led us to revisit the structural requirement for this editing by mutagenesis analysis. In vivo nuclear injection experiments of mutated editing substrates demonstrate that a non-conserved structure is a determinant for editing. This structure contains bulges either on the same or the strand opposing the edited adenosine. The position of these bulges and the distance to the edited base regulate editing. Moreover, elevated folding temperature can lead to a switch in RNA editing suggesting an RNA structural change. Our results indicate the importance of RNA tertiary structure in determining RNA editing.'
doi: 10.1093/nar/gkr144
thumb_url: 'http://europepmc.org/articles/PMC3141254/bin/gkr144f7.gif'
figure_no: 7
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC3141254/bin/gkr144f7.jpg' style='max-height: 300px'>
### Figure 7
<p style='font-size: 10px;'>Structural probing of editing substrates *in vitro*. Left panels: representative primer extension gels showing the *in vitro* modification pattern of the GABRA editing substrates of *H. sapiens* WT (**A**), *H. sapiens* mutant C40 (**B**), *Mus musculus* WT (**C**), *Mus musculus* mutant C40 (**D**), *Gallus gallus* WT (the sequence of Gabra1; **E**) and *Gallus gallus* mutant 14 (**F**). A, G, U and C denote sequencing lanes. In the lane ‘–’, natural stops of the reverse transcriptase are shown to detect natural stops of the extension (independent of DMS, CMCT or Kethoxal modification; RNA was folded, but not incubated with either reagent). Lanes labelled DMS: the *in vitro* DMS modification pattern is shown. DMS methylates A-N1 and C-N3, if they are not involved in H-bonding. Lanes labelled Kethoxal: the *in vitro* Kethoxal modification pattern is shown. Kethoxal modifies G-N1 and G-N2, if they are not involved in H-bonding. Lanes labelled CMCT: the *in vitro* CMCT modification pattern is shown. CMCT modifies G-N1 and U-N3, if they are not involved in H-bonding. Right panels: Modifications were plotted onto the secondary-structure maps of respective editing substrates. Red filled circles indicate the detected modification patterns, whereby the size of the filled circles correlates with the relative modification intensity of individual bases. Residues boxed in grey (**B–F**) indicate base changes relative to the *H. sapiens* WT editing substrate (**A**).</p>