---
name: 21232099_fig1
title: An overview to the investigative approach to species testing in wildlife forensic science.
authorString: 'Linacre A, Tobe SS.'
pmcid: PMC3032691
pmid: '21232099'
abstract: ' The extent of wildlife crime is unknown but it is on the increase and has observable effects with the dramatic decline in many species of flora and fauna. The growing awareness of this area of criminal activity is reflected in the increase in research papers on animal DNA testing, either for the identification of species or for the genetic linkage of a sample to a particular organism. This review focuses on the use of species testing in wildlife crime investigations. Species identification relies primarily on genetic loci within the mitochondrial genome; focusing on the cytochrome b and cytochrome oxidase 1 genes. The use of cytochrome b gained early prominence in species identification through its use in taxonomic and phylogenetic studies, while the gene sequence for cytochrome oxidase was adopted by the Barcode for Life research group. This review compares how these two loci are used in species identification with respect to wildlife crime investigations. As more forensic science laboratories undertake work in the wildlife area, it is important that the quality of work is of the highest standard and that the conclusions reached are based on scientific principles. A key issue in reporting on the identification of a particular species is a knowledge of both the intraspecies variation and the possible overlap of sequence variation from one species to that of a closely related species. Recent data showing this degree of genetic separation in mammalian species will allow greater confidence when preparing a report on an alleged event where the identification of the species is of prime importance. The aim of this review is to illustrate aspects of species testing in wildlife forensic science and to explain how a knowledge of genetic variation at the genus and species level can aid in the reporting of results.'
doi: 10.1186/2041-2223-2-2
thumb_url: 'http://europepmc.org/articles/PMC3032691/bin/2041-2223-2-2-1.gif'
figure_no: 1
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC3032691/bin/2041-2223-2-2-1.jpg' style='max-height: 300px'>
### Figure 1
<p style='font-size: 10px;'>**Figure 1a and b showing a small part of the cyt b gene DNA sequence and their differences for four species**Figure 1a shows bases 121 â€“ 241 of the cyt b gene for the Tibetan antelope (Pantholops hodgsonii: accession number <ext-link ext-link-type="gen" xlink:href="AF034724">AF034724</ext-link>) compared to the mammalian species with the closest homologies to this part of the sequence; being sheep (Ovis aries: accession <ext-link ext-link-type="gen" xlink:href="AB0068000">AB0068000</ext-link>), the Pyrenean Chamoix  (Rupicapra pyrenaica: accession number <ext-link ext-link-type="gen" xlink:href="AF034726">AF034726</ext-link>) and a goat (Capra sumatrensis: accession number <ext-link ext-link-type="gen" xlink:href="AY669321">AY669321</ext-link>).  The unknown (or questioned) sequence comes from a shawl suspected as being from Shatoosh and derived from the Tibetan Antelope.  Figure 1b shows the number of bases differing between the four species (bottom of the rectangle) and the % similarity over the 120 bases. It would be normal to use over 400 bases in a similarity search rather than only the 120 as shown above but this indicates the process used in species testing.</p>