---
name: 24342523_fig3
title: Population genomics of the endangered giant Galapagos tortoise.
authorString: 'Loire E, Chiari Y, Bernard A, Cahais V, Romiguier J, Nabholz B, Lourenço JM, Galtier N.'
pmcid: PMC4053747
pmid: '24342523'
abstract: 'BACKGROUND: The giant Galápagos tortoise, Chelonoidis nigra, is a large-sized terrestrial chelonian of high patrimonial interest. The species recently colonized a small continental archipelago, the Galápagos Islands, where it has been facing novel environmental conditions and limited resource availability. To explore the genomic consequences of this ecological shift, we analyze the transcriptomic variability of five individuals of C. nigra, and compare it to similar data obtained from several continental species of turtles. RESULTS: Having clarified the timing of divergence in the Chelonoidis genus, we report in C. nigra a very low level of genetic polymorphism, signatures of a weakened efficacy of purifying selection, and an elevated mutation load in coding and regulatory sequences. These results are consistent with the hypothesis of an extremely low long-term effective population size in this insular species. Functional evolutionary analyses reveal a reduced diversity of immunity genes in C. nigra, in line with the hypothesis of attenuated pathogen diversity in islands, and an increased selective pressure on genes involved in response to stress, potentially related to the climatic instability of its environment and its elongated lifespan. Finally, we detect no population structure or homozygosity excess in our five-individual sample. CONCLUSIONS: These results enlighten the molecular evolution of an endangered taxon in a stressful environment and point to island endemic species as a promising model for the study of the deleterious effects on genome evolution of a reduced long-term population size.'
doi: 10.1186/gb-2013-14-12-r136
thumb_url: 'http://europepmc.org/articles/PMC4053747/bin/gb-2013-14-12-r136-3.gif'
figure_no: 3
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC4053747/bin/gb-2013-14-12-r136-3.jpg' style='max-height: 300px'>
### Figure 3
<p style='font-size: 10px;'>**Flanking *****vs*****. coding region diversity in *****C. nigra *****and *****E. orbicularis*****.** π<sub>S</sub>: average nucleotide diversity at synonymous sites; π<sub>5</sub>: average nucleotide diversity at 5′-UTR sites; π<sub>3</sub>: average nucleotide diversity at 3′-UTR sites. Very similar results were obtained when we only analyzed predicted cDNAs for which both coding sequence and 3′ UTR sequence were available, and when we restricted the analysis to the 500 first bases of the 3′ UTR regions.</p>