---
name: 25342462_fig1
title: Global distribution of Chelonid fibropapilloma-associated herpesvirus among clinically healthy sea turtles.
authorString: 'Alfaro-Núñez A, Frost Bertelsen M, Bojesen AM, Rasmussen I, Zepeda-Mendoza L, Tange Olsen M, Gilbert MT.'
pmcid: PMC4219010
pmid: '25342462'
abstract: 'BACKGROUND: Fibropapillomatosis (FP) is a neoplastic disease characterized by cutaneous tumours that has been documented to infect all sea turtle species. Chelonid fibropapilloma-associated herpesvirus (CFPHV) is believed to be the aetiological agent of FP, based principally on consistent PCR-based detection of herpesvirus DNA sequences from FP tumours. We used a recently described PCR-based assay that targets 3 conserved CFPHV genes, to survey 208 green turtles (Chelonia mydas). This included both FP tumour exhibiting and clinically healthy individuals. An additional 129 globally distributed clinically healthy individual sea turtles; representing four other species were also screened. RESULTS: CFPHV DNA sequences were obtained from 37/37 (100%) FP exhibiting green turtles, and 45/300 (15%) clinically healthy animals spanning all five species. Although the frequency of infected individuals per turtle population varied considerably, most global populations contained at least one CFPHV positive individual, with the exception of various turtle species from the Arabian Gulf, Northern Indian Ocean and Puerto Rico. Haplotype analysis of the different gene markers clustered the CFPHV DNA sequences for two of the markers (UL18 and UL22) in turtles from Turks and Caicos separate to all others, regardless of host species or geographic origin. CONCLUSION: Presence of CFPHV DNA within globally distributed samples for all five species of sea turtle was confirmed. While 100% of the FP exhibiting green turtles yielded CFPHV sequences, surprisingly, so did 15% of the clinically healthy turtles. We hypothesize that turtle populations with zero (0%) CFPHV frequency may be attributed to possible environmental differences, diet and/or genetic resistance in these individuals. Our results provide first data on the prevalence of CFPHV among seemingly healthy turtles; a factor that may not be directly correlated to the disease incidence, but may suggest of a long-term co-evolutionary latent infection interaction between CFPHV and its turtle-host across species. Finally, computational analysis of amino acid variants within the Turks and Caicos samples suggest potential functional importance in a substitution for marker UL18 that encodes the major capsid protein gene, which potentially could explain differences in pathogenicity. Nevertheless, such a theory remains to be validated by further research.'
doi: 10.1186/s12862-014-0206-z
thumb_url: 'http://europepmc.org/articles/PMC4219010/bin/12862_2014_206_Fig1_HTML.gif'
figure_no: 1
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC4219010/bin/12862_2014_206_Fig1_HTML.jpg' style='max-height: 300px'>
### Figure 1
<p style='font-size: 10px;'>**CFPHV global detection, haplotype networks and AA protein structures. (A)** Global CFPHV detection given as proportion of each turtle species’ populations. Different colours represent category of tissue sample and infectivity status as follows; Orange = FP exhibiting turtles found CFPHV positive, Yellow = clinically healthy turtles also CFPHV positive, and Green = clinically healthy turtles CFPHV negative (CFPHV free), **(B)** Percentage of CFPHV turtle infectivity grouped by different turtle species (same colour code as panel **A**), **(C)** Regional haplotype networks for markers UL18, UL22 and UL27 clustered by regional sample origins, and showing the respective number of haplotypes found per marker, and **(D)** Amino acid radical substitution-predicted protein structure models for marker UL18, where globally distributed samples have the reference structure, while green turtles from Turks and Caicos, plus one green turtle from Portugal correspond to the alternative variant structure.</p>