---
name: 19473484_fig4
title: Can comprehensive background knowledge be incorporated into substitution models to improve phylogenetic analyses? A case study on major arthropod relationships.
authorString: "von Reumont BM, Meusemann K, Szucsich NU, Dell'Ampio E, Gowri-Shankar V, Bartel D, Simon S, Letsch HO, Stocsits RR, Luan YX, WÃ¤gele JW, Pass G, Hadrys H, Misof B."
pmcid: PMC2695459
pmid: '19473484'
abstract: ' BACKGROUND: Whenever different data sets arrive at conflicting phylogenetic hypotheses, only testable causal explanations of sources of errors in at least one of the data sets allow us to critically choose among the conflicting hypotheses of relationships. The large (28S) and small (18S) subunit rRNAs are among the most popular markers for studies of deep phylogenies. However, some nodes supported by this data are suspected of being artifacts caused by peculiarities of the evolution of these molecules. Arthropod phylogeny is an especially controversial subject dotted with conflicting hypotheses which are dependent on data set and method of reconstruction. We assume that phylogenetic analyses based on these genes can be improved further i) by enlarging the taxon sample and ii) employing more realistic models of sequence evolution incorporating non-stationary substitution processes and iii) considering covariation and pairing of sites in rRNA-genes. RESULTS: We analyzed a large set of arthropod sequences, applied new tools for quality control of data prior to tree reconstruction, and increased the biological realism of substitution models. Although the split-decomposition network indicated a high noise content in the data set, our measures were able to both improve the analyses and give causal explanations for some incongruities mentioned from analyses of rRNA sequences. However, misleading effects did not completely disappear. CONCLUSION: Analyses of data sets that result in ambiguous phylogenetic hypotheses demand for methods, which do not only filter stochastic noise, but likewise allow to differentiate phylogenetic signal from systematic biases. Such methods can only rely on our findings regarding the evolution of the analyzed data. Analyses on independent data sets then are crucial to test the plausibility of the results. Our approach can easily be extended to genomic data, as well, whereby layers of quality assessment are set up applicable to phylogenetic reconstructions in general.'
doi: 10.1186/1471-2148-9-119
thumb_url: 'http://europepmc.org/articles/PMC2695459/bin/1471-2148-9-119-4.gif'
figure_no: 4
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2695459/bin/1471-2148-9-119-4.jpg' style='max-height: 300px'>
### Figure 4
<p style='font-size: 10px;'>**Time-homogeneous consensus tree**. Consensus tree from 18,000 sampled trees of the time-homogeneous substitution process inferred by *PHASE-2.0*, graphically processed with Adobe Illustrator CS2. Support values below 0.70 are not shown (nodes without dots), nodes with a maximum posterior probability (pP) of 1.0 are represented by dots only. The grey dot indicates the clade containing all hexapod taxa including *Hutchinsoniella *(Crustacea) + *Lepisma *(Zygentoma); its node value is pP 0.58. Quotation marks indicate that monophyly is not supported in the given tree.</p>