---
name: 18798978_fig2
title: 'Careful with understudied phyla: the case of chaetognath.'
authorString: 'Marlétaz F, Le Parco Y.'
pmcid: PMC2566580
pmid: '18798978'
abstract: " BACKGROUND: A recent study by Barthélémy et al. described a set of ribosomal protein (RP) genes extracted from a collection of expressed sequence tags (ESTs) of the chaetognath (arrow worm) Spadella cephaloptera. Three main conclusions were drawn in this paper. First, the authors stated that RP genes present paralogous copies, which have arisen through allopolyploidization. Second, they reported two alternate nucleotide stretches conserved within the 5' untranslated regions (UTR) of multiple ribosomal cDNAs and they suggested that these motifs are involved in the differential transcriptional regulation of paralogous RP genes. Third, they claimed that the phylogenetic position of chaetognaths could not be accurately inferred from a RP dataset because of the persistence of two problems: a long branch attraction (LBA) artefact and a compositional bias. RESULTS: We reconsider here the results described in Barthélémy et al. and question the evidence on which they are based. We find that their evidence for paralogous copies relies on faulty PCR experiments since they attempted to amplify DNA fragments absent from the genomic template. Our PCR experiments proved that the conserved motifs in 5'UTRs that they targeted in their amplifications are added post-transcriptionally by a trans-splicing mechanism. Then, we showed that the lack of phylogenetic resolution observed by these authors is due to limited taxon sampling and not to LBA or to compositional bias. A ribosomal protein dataset thus fully supports the position of chaetognaths as sister group of all other protostomes. This reinterpretation demonstrates that the statements of Barthélémy et al. should be taken with caution because they rely on inaccurate evidence. CONCLUSION: The genomic study of an unconventional model organism is a meaningful approach to understand the evolution of animals. However, the previous study came to incorrect conclusions on the basis of experiments that omitted validation procedures."
doi: 10.1186/1471-2148-8-251
thumb_url: 'http://europepmc.org/articles/PMC2566580/bin/1471-2148-8-251-2.gif'
figure_no: 2
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2566580/bin/1471-2148-8-251-2.jpg' style='max-height: 300px'>
### Figure 2
<p style='font-size: 10px;'>**Ribosomal protein dataset fully resolves animal phylogeny and reveals chaetognaths to be the sister taxon of all other protostomes.** This tree was inferred using phylobayes with the CAT model from a dataset of 12,764 amino acid positions after removal of ambiguous positions by Gblock. Bayesian posterior probabilities (upper numbers, bold type) and ML bootstrap support (lower numbers, italic type) are indicated as support values and circled for nodes relevant to chaetognath branching. Bayesian posterior probabilities over 95% and ML-bootstrap values over 70% are considered valid support.</p>