---
name: 20226016_fig2
title: Transcriptome survey of the anhydrobiotic tardigrade Milnesium tardigradum in comparison with Hypsibius dujardini and Richtersius coronifer.
authorString: 'Mali B, Grohme MA, Förster F, Dandekar T, Schnölzer M, Reuter D, Wełnicz W, Schill RO, Frohme M.'
pmcid: PMC2848246
pmid: '20226016'
abstract: ' BACKGROUND: The phenomenon of desiccation tolerance, also called anhydrobiosis, involves the ability of an organism to survive the loss of almost all cellular water without sustaining irreversible damage. Although there are several physiological, morphological and ecological studies on tardigrades, only limited DNA sequence information is available. Therefore, we explored the transcriptome in the active and anhydrobiotic state of the tardigrade Milnesium tardigradum which has extraordinary tolerance to desiccation and freezing. In this study, we present the first overview of the transcriptome of M. tardigradum and its response to desiccation and discuss potential parallels to stress responses in other organisms. RESULTS: We sequenced a total of 9984 expressed sequence tags (ESTs) from two cDNA libraries from the eutardigrade M. tardigradum in its active and inactive, anhydrobiotic (tun) stage. Assembly of these ESTs resulted in 3283 putative unique transcripts, whereof approximately 50% showed significant sequence similarity to known genes. The resulting unigenes were functionally annotated using the Gene Ontology (GO) vocabulary. A GO term enrichment analysis revealed several GOs that were significantly underrepresented in the inactive stage. Furthermore we compared the putative unigenes of M. tardigradum with ESTs from two other eutardigrade species that are available from public sequence databases, namely Richtersius coronifer and Hypsibius dujardini. The processed sequences of the three tardigrade species revealed similar functional content and the M. tardigradum dataset contained additional sequences from tardigrades not present in the other two. CONCLUSIONS: This study describes novel sequence data from the tardigrade M. tardigradum, which significantly contributes to the available tardigrade sequence data and will help to establish this extraordinary tardigrade as a model for studying anhydrobiosis. Functional comparison of active and anhydrobiotic tardigrades revealed a differential distribution of Gene Ontology terms associated with chromatin structure and the translation machinery, which are underrepresented in the inactive animals. These findings imply a widespread metabolic response of the animals on dehydration. The collective tardigrade transcriptome data will serve as a reference for further studies and support the identification and characterization of genes involved in the anhydrobiotic response.'
doi: 10.1186/1471-2164-11-168
thumb_url: 'http://europepmc.org/articles/PMC2848246/bin/1471-2164-11-168-2.gif'
figure_no: 2
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2848246/bin/1471-2164-11-168-2.jpg' style='max-height: 300px'>
### Figure 2
<p style='font-size: 10px;'>**Differentially represented Gene Ontology terms between active and inactive EST libraries of *M. tardigradum***. GO enrichment analysis between active and inactive EST libraries using Fisher's exact test with a false discovery rate (FDR) cutoff of p ≤ 0.05 (*) and p ≤ 0.01 (**). The number of transcripts associated with a specific GO term are represented as percentage of all functionally annotated EST in their respective libraries (active: N = 1207 and inactive: N = 1055).</p>