---
name: 18447908_fig1
title: 'Assessing meiofaunal variation among individuals utilising morphological and molecular approaches: an example using the Tardigrada.'
authorString: 'Sands CJ, Convey P, Linse K, McInnes SJ.'
pmcid: PMC2387140
pmid: '18447908'
abstract: ' BACKGROUND: Meiofauna - multicellular animals captured between sieve size 45 mum and 1000 mum - are a fundamental component of terrestrial, and marine benthic ecosystems, forming an integral element of food webs, and playing a critical roll in nutrient recycling. Most phyla have meiofaunal representatives and studies of these taxa impact on a wide variety of sub-disciplines as well as having social and economic implications. However, studies of variation in meiofauna are presented with several important challenges. Isolating individuals from a sample substrate is a time consuming process, and identification requires increasingly scarce taxonomic expertise. Finding suitable morphological characters in many of these organisms is often difficult even for experts. Molecular markers are extremely useful for identifying variation in morphologically conserved organisms. However, for many species markers need to be developed de novo, while DNA can often only be extracted from pooled samples in order to obtain sufficient quantity and quality. Importantly, multiple independent markers are required to reconcile gene evolution with species evolution. In this primarily methodological paper we provide a proof of principle of a novel and effective protocol for the isolation of meiofauna from an environmental sample. We also go on to illustrate examples of the implications arising from subsequent screening for genetic variation at the level of the individual using ribosomal, mitochondrial and single copy nuclear markers. RESULTS: To isolate individual tardigrades from their habitat substrate we used a non-toxic density gradient media that did not interfere with downstream biochemical processes. Using a simple DNA release technique and nested polymerase chain reaction with universal primers we were able amplify multi-copy and, to some extent, single copy genes from individual tardigrades. Maximum likelihood trees from ribosomal 18S, mitochondrial cytochrome oxidase subunit 1, and the single copy nuclear gene Wingless support a recent study indicating that the family Hypsibiidae is a non-monophyletic group. From these sequences we were able to detect variation between individuals at each locus that allowed us to identify the presence of cryptic taxa that would otherwise have been overlooked. CONCLUSION: Molecular results obtained from individuals, rather than pooled samples, are a prerequisite to enable levels of variation to be placed into context. In this study we have provided a proof of principle of this approach for meiofaunal tardigrades, an important group of soil biota previously not considered amenable to such studies, thereby paving the way for more comprehensive phylogenetic studies using multiple nuclear markers, and population genetic studies.'
doi: 10.1186/1472-6785-8-7
thumb_url: 'http://europepmc.org/articles/PMC2387140/bin/1472-6785-8-7-1.gif'
figure_no: 1
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2387140/bin/1472-6785-8-7-1.jpg' style='max-height: 300px'>
### Figure 1
<p style='font-size: 10px;'>**Tardigrade phylogeny constructed from 18S rRNA**. Phylogeny produced using maximum likelihood analyses under a GTR+I+Γ model using 884 bases of *18S *rRNA. Numbers at nodes are support values generated from 1000 bootstrap pseudoreplicates. Terminal labels associated with unique numbers were identified as: Macro 001–008 – *Macrobiotus furciger*, Macro 048-52,68-077 – *Macrobiotus sp*. (hufelandi type), Dacty – *Dactylobiotus sp*., Acut – *Acutuncus antarcticus*, Diph – *Diphascon sp*., RamO – *Ramazzottius oberhaeuseri*, Isohyp – *Isohypsibius asper*, MilnT – *Milnesium tardigradum*, MilnC – *Milnesium sp. *"charcot", Bryod – *Bryodelphax sp*., EchinC – *Echiniscus sp*., EchinT – *Echiniscus testudo*.</p>