---
name: 19555513_fig4
title: 'Phylogenetic support values are not necessarily informative: the case of the Serialia hypothesis (a mollusk phylogeny).'
authorString: 'Wägele JW, Letsch H, Klussmann-Kolb A, Mayer C, Misof B, Wägele H.'
pmcid: PMC2710323
pmid: '19555513'
abstract: " BACKGROUND: Molecular phylogenies are being published increasingly and many biologists rely on the most recent topologies. However, different phylogenetic trees often contain conflicting results and contradict significant background data. Not knowing how reliable traditional knowledge is, a crucial question concerns the quality of newly produced molecular data. The information content of DNA alignments is rarely discussed, as quality statements are mostly restricted to the statistical support of clades. Here we present a case study of a recently published mollusk phylogeny that contains surprising groupings, based on five genes and 108 species, and we apply new or rarely used tools for the analysis of the information content of alignments and for the filtering of noise (masking of random-like alignment regions, split decomposition, phylogenetic networks, quartet mapping). RESULTS: The data are very fragmentary and contain contaminations. We show that that signal-like patterns in the data set are conflicting and partly not distinct and that the reported strong support for a \"rather surprising result\" (monoplacophorans and chitons form a monophylum Serialia) does not exist at the level of primary homologies. Split-decomposition, quartet mapping and neighbornet analyses reveal conflicting nucleotide patterns and lack of distinct phylogenetic signal for the deeper phylogeny of mollusks. CONCLUSION: Even though currently a majority of molecular phylogenies are being justified with reference to the 'statistical' support of clades in tree topologies, this confidence seems to be unfounded. Contradictions between phylogenies based on different analyses are already a strong indication of unnoticed pitfalls. The use of tree-independent tools for exploratory analyses of data quality is highly recommended. Concerning the new mollusk phylogeny more convincing evidence is needed."
doi: 10.1186/1742-9994-6-12
thumb_url: 'http://europepmc.org/articles/PMC2710323/bin/1742-9994-6-12-4.gif'
figure_no: 4
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC2710323/bin/1742-9994-6-12-4.jpg' style='max-height: 300px'>
### Figure 4
<p style='font-size: 10px;'>**Spectrum of split-supporting positions as estimated with SAMS**. Column height represents the number of clade-supporting positions, i.e. putative primary homologies. Column parts above the y axis represent the best supported of the two groups of a split, the part below the axis corresponds to the second group. Red: asymmetrical positions (conserved character state only in functional ingroup); yellow: noisy positions (more than one character state in functional in- and outgroup; see Wägele and Rödding 1998a, b). The first ten columns represent the best supported splits and contain the following groupings: 1: *Cellana sp.+ Eulepetopsis vitrea*; 2: Coleoida; 3 Coleoida + *Creseis sp*; 4: *Nautilus pompilius *+ *Nautilus scrobiculatus*; 5: Cephalopoda; 6: Cephalopoda + *Creseis sp*.; 7: *Paranerilla limicola *+ *Chaetoderma nitidulum*; 8: *Chamelea striatula *+ *Corbicula spp *+ *Mercenaria mercenaria*; 9: *Cellana sp.+ Eulepetopsis vitrea *+ Coleoida; 10: *Cellana sp*. + *Eulepetopsis vitrea *+ *Creseis sp*. + *Laevipilina antarctica*. No split was detected that unites *Laevipilina *and Polyplacophora.</p>