---
name: 25216354_fig4
title: Comparative genome sequencing reveals genomic signature of extreme desiccation tolerance in the anhydrobiotic midge.
authorString: 'Gusev O, Suetsugu Y, Cornette R, Kawashima T, Logacheva MD, Kondrashov AS, Penin AA, Hatanaka R, Kikuta S, Shimura S, Kanamori H, Katayose Y, Matsumoto T, Shagimardanova E, Alexeev D, Govorun V, Wisecaver J, Mikheyev A, Koyanagi R, Fujie M, Nishiyama T, Shigenobu S, Shibata TF, Golygina V, Hasebe M, Okuda T, Satoh N, Kikawada T.'
pmcid: PMC4175575
pmid: '25216354'
abstract: 'Anhydrobiosis represents an extreme example of tolerance adaptation to water loss, where an organism can survive in an ametabolic state until water returns. Here we report the first comparative analysis examining the genomic background of extreme desiccation tolerance, which is exclusively found in larvae of the only anhydrobiotic insect, Polypedilum vanderplanki. We compare the genomes of P. vanderplanki and a congeneric desiccation-sensitive midge P. nubifer. We determine that the genome of the anhydrobiotic species specifically contains clusters of multi-copy genes with products that act as molecular shields. In addition, the genome possesses several groups of genes with high similarity to known protective proteins. However, these genes are located in distinct paralogous clusters in the genome apart from the classical orthologues of the corresponding genes shared by both chironomids and other insects. The transcripts of these clustered paralogues contribute to a large majority of the mRNA pool in the desiccating larvae and most likely define successful anhydrobiosis. Comparison of expression patterns of orthologues between two chironomid species provides evidence for the existence of desiccation-specific gene expression systems in P. vanderplanki.'
doi: 10.1038/ncomms5784
thumb_url: 'http://europepmc.org/articles/PMC4175575/bin/ncomms5784-f4.gif'
figure_no: 4
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC4175575/bin/ncomms5784-f4.jpg' style='max-height: 300px'>
### Figure 4
<p style='font-size: 10px;'><title>Evolutionary relationships of the classical and novel desiccation-responsive TRX and PIMT proteins.</title> (**a**) Phylogenetic tree of TRX proteins showing the clusters of classical insect TRX-1 (green), TRX-2 (grey), TRX-3 (blue) and the cluster of desiccation-responsive TRX, specific to *P. vanderplanki* (pink). (**b**) Phylogenetic tree of PIMT proteins showing the cluster of the classical PIMT-1 conserved among Diptera (green) and the cluster of desiccation-responsive PIMT proteins, which are specific to *P. vanderplanki* (pink). The evolutionary history was inferred using the neighbor-joining method and the evolutionary distances were computed using the maximum likelihood estimation (units: amino-acid substitutions per site). Pv, *P. vanderplanki*; Pn, *P. nubifer*; Aa, *Aedes aegypti*; Ag, *Anopheles gambiae*; Am, *Apis mellifera*; Cq, *Culex quinquefasciatus*; At, *Arabidopsis thaliana*; Dm, *Drosophila melanogaster*; Hs, *Homo sapiens*; Mm, *Mus musculus*; Nv, *Nasonia vitripennis*; XI, *Xenopus laevis*.</p>