---
name: 25216354_fig2
title: Comparative genome sequencing reveals genomic signature of extreme desiccation tolerance in the anhydrobiotic midge.
authorString: 'Gusev O, Suetsugu Y, Cornette R, Kawashima T, Logacheva MD, Kondrashov AS, Penin AA, Hatanaka R, Kikuta S, Shimura S, Kanamori H, Katayose Y, Matsumoto T, Shagimardanova E, Alexeev D, Govorun V, Wisecaver J, Mikheyev A, Koyanagi R, Fujie M, Nishiyama T, Shigenobu S, Shibata TF, Golygina V, Hasebe M, Okuda T, Satoh N, Kikawada T.'
pmcid: PMC4175575
pmid: '25216354'
abstract: 'Anhydrobiosis represents an extreme example of tolerance adaptation to water loss, where an organism can survive in an ametabolic state until water returns. Here we report the first comparative analysis examining the genomic background of extreme desiccation tolerance, which is exclusively found in larvae of the only anhydrobiotic insect, Polypedilum vanderplanki. We compare the genomes of P. vanderplanki and a congeneric desiccation-sensitive midge P. nubifer. We determine that the genome of the anhydrobiotic species specifically contains clusters of multi-copy genes with products that act as molecular shields. In addition, the genome possesses several groups of genes with high similarity to known protective proteins. However, these genes are located in distinct paralogous clusters in the genome apart from the classical orthologues of the corresponding genes shared by both chironomids and other insects. The transcripts of these clustered paralogues contribute to a large majority of the mRNA pool in the desiccating larvae and most likely define successful anhydrobiosis. Comparison of expression patterns of orthologues between two chironomid species provides evidence for the existence of desiccation-specific gene expression systems in P. vanderplanki.'
doi: 10.1038/ncomms5784
thumb_url: 'http://europepmc.org/articles/PMC4175575/bin/ncomms5784-f2.gif'
figure_no: 2
tags:
  - eupmc
  - figure
---
<img src='http://europepmc.org/articles/PMC4175575/bin/ncomms5784-f2.jpg' style='max-height: 300px'>
### Figure 2
<p style='font-size: 10px;'><title>Putative mechanism for the evolution of ARId in the *P. vanderplanki* genome.</title> ARIds are genomic regions containing clusters of duplicated genes that are transcriptionally active during anhydrobiosis. (**a**) A gene of foreign origin (for example, LEA protein) is incorporated into *P. vanderplanki* genome by HGT and undergoes extensive duplications and shuffling. (**b**) A pre-existing *P. vanderplanki* gene originally not involved in anhydrobiosis and originating from another region of *P. vanderplanki* genome was inserted to a new locus by intragenomic duplication (IGD) and undergoes extensive duplications and shuffling to acquire or improve a specific function for desiccation tolerance. All the genes in the ARIds from both **a**,**b** become highly upregulated during anhydrobiosis (red arrows).</p>